contracts/votingMachines/VotingMachineCallbacks.sol

```diff
@@ -3,32 +3,24 @@ pragma solidity ^0.5.17;
 import "@daostack/infra-experimental/contracts/votingMachines/GenesisProtocol.sol";
 import "../controller/Avatar.sol";
 import "../controller/Controller.sol";
+import "../schemes/ArcScheme.sol";
 
 
-contract VotingMachineCallbacks is VotingMachineCallbacksInterface {
-
-    struct ProposalInfo {
-        uint256 blockNumber; // the proposal's block number
-        Avatar avatar; // the proposal's avatar
-    }
+contract VotingMachineCallbacks is VotingMachineCallbacksInterface, ArcScheme {
 
     modifier onlyVotingMachine(bytes32 _proposalId) {
-        require(proposalsInfo[msg.sender][_proposalId].avatar != Avatar(address(0)), "only VotingMachine");
+        require(proposalsBlockNumber[msg.sender][_proposalId] != 0, "only VotingMachine");
         _;
     }
 
-    // VotingMaching  ->  proposalId  ->  ProposalInfo
-    mapping(address => mapping(bytes32 => ProposalInfo)) public proposalsInfo;
+    // VotingMaching  ->  proposalId  ->  blockNumber
+    mapping(address => mapping(bytes32 => uint256)) public proposalsBlockNumber;
```

Comment:
how about do it mapping between proposalId to blockNumber.
there is a single voting machine for each scheme,
- we can add votingMachine+voteParams to `ArcScheme` and init that to zero if there is no voting machine.not sure..